### DISCUSSION TITLE
Is it possible to run format and check in the same command?

### DISCUSSION THREAD

**Discussion by bryanwweber** [astral-sh/ruff#8515] [Q&A] [answered]
I currently have two CLI commands configured, one that runs `ruff format` and one that runs `ruff check --fix`. Is it possible to combine these into one command? Running the obvious options of `ruff format check --fix src/` says that `--fix` isn't valid for `ruff format` and `ruff check --fix format src/` complains that `format` isn't a file or directory. Thank you!

**Comment by zanieb** [comment, marked as answer]
Please see https://github.com/astral-sh/ruff/issues/8232

**Reply by bryanwweber** [reply]
Thanks for the link! Somehow my Google-fu failed at finding that issue.

**Reply by zanieb** [reply]
No problem :)
